extensions/tldr/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "tldr",
+  "title": "tldr",
+  "author": "pomdtr",
+  "license":"MIT",
+  "description": "Search tldr Pages",
+  "icon": "tldr.png",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search TLDR Pages",
+      "subtitle": "tldr",
+      "description": "Search Commands Cheatsheets.",
+      "mode": "view"
+    },
+    {
+      "name": "refresh-pages",
+      "title": "Refresh TLDR Pages",
```

Comment:
```suggestion
      "title": "Refresh Pages",
```

Redundant, already added to the subtitle — Could also be added as a ActionPanel.item in “Search Pages”

Reply:
I'll consider it from another PR :)

Reply:
Nevermind, I implemented it